layouts/partials/docs/hero.html

```diff
@@ -73,7 +73,7 @@
               </span>
               {{ end }}
             </a>
-            {{ else if ne "/v2.3/" $new_version }}
+            {{ else if ((ne "/v2.3/" $new_version) | and (ne "/v3.1/" $new_version)) }}
```

Comment:
Maybe in the next round of changes we could you a regex-match test here (and in the nav-panel.html file)?

Reply:
That's sortof how i'm doing it in the `docsy` branch.

Reply:
https://github.com/etcd-io/website/blob/bbc8bf14e3328400e00c75f94737594b5b434144/layouts/partials/navbar-version-selector.html#L22-L38